gui-src/js/main.js

```diff
@@ -1,6 +1,11 @@
 import { licenseText } from './license.js';
 
-const { invoke } = window.__TAURI__.core;
+if (window.__TAURI__) {
```

Comment:
Sorry, but this change is causing an issue when trying to load world due to `invoke` now being scoped only inside the `if` block. You'll need to create a `let` variable above (in the global scope), for example, and assign it the value of this `invoke`.

<img width="1012" alt="image" src="https://github.com/user-attachments/assets/3cdeb56d-3040-4c76-bf82-7c285d271349" />
(screenshot after changing the code a tiny bit to display the `errorCode`)

@TheComputerGuy96 @louis-e

Reply:
#296 should probably fix this